http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/NettyRequestExecutor.java

```diff
@@ -196,6 +197,7 @@ private void configureChannel() {
         channel.attr(REQUEST_CONTEXT_KEY).set(context);
         channel.attr(RESPONSE_COMPLETE_KEY).set(false);
         channel.attr(LAST_HTTP_CONTENT_RECEIVED_KEY).set(false);
+        channel.attr(RESPONSE_100_CONTINUE_MESSAGE).set(false);
```

Comment:
Would it be possible to mark the completion of request in `HttpStreamHandler` and get rid of LastHttpContentHandler? That way, we don't need to handle 100 continue message this classe

Reply:
Raised https://github.com/aws/aws-sdk-java-v2/pull/3535/ to change the logic where LAST_HTTP_CONTENT_RECEIVED_KEY is set